modules/ROOT/partials/configuration/convert_urls.adoc

```diff
@@ -1,6 +1,7 @@
-== convert_urls
+[[convert_urls]]
+== `convert_urls`
 
-This option enables you to control whether {productname} is to be smart and restore URLs to their original values. URLs are automatically converted (messed up) by default because the browser's built-in logic works this way. There is no way to get the real URL unless you store it away. If you set this option to `false` it tries to keep these URLs intact. This option is set to `true` by default, which means URLs are forced to be either *absolute* or *relative* depending on the state of <<relative_urls,relative_urls>>.
+This option enables you to control whether {productname} is to be smart and restore URLs to their original values. URLs are automatically converted (messed up) by default because the browser's built-in logic works this way. There is no way to get the real URL unless you store it away. If you set this option to `false` it tries to keep these URLs intact. This option is set to `true` by default, which means URLs are forced to be either *absolute* or *relative* depending on the state of xref:relative_urls[relative_urls].
```

Comment:
The `relative_urls` ID isn't in this page so we probably should link it to the actual page otherwise this link could break if used elsewhere.
```suggestion
This option enables you to control whether {productname} is to be smart and restore URLs to their original values. URLs are automatically converted (messed up) by default because the browser's built-in logic works this way. There is no way to get the real URL unless you store it away. If you set this option to `false` it tries to keep these URLs intact. This option is set to `true` by default, which means URLs are forced to be either *absolute* or *relative* depending on the state of xref:configure/url-handling.adoc#relative_urls[relative_urls].
```

Reply:
done